### ISSUE TITLE
508-defect-2 [FOCUS]: Focus should be sent to the `h1` upon closing the modal

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#30029]
#### VFS team name
eBenefits

#### VFS product name
Dependency Verification

#### Point of Contact/Reviewers
Josh Kim (Accessibility)

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Upon closing the modal, focus should not be sent to the skip to content link or breadcrumb.

### Link, screenshot or steps to recreate

<img width="1250" alt="close-modal-focus" src="https://user-images.githubusercontent.com/14154792/133645615-5d2c9b88-e3dc-4c54-83ed-96b6e1a55a5a.png">

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
Focus should be sent to the `h1` by giving the `h1` a `tabindex="-1"`. By doing so, screen reader users will immediately know what page they are on e.g. "Your VA Dependents" instead of having to listen to a long breadcrumb e.g. "Breadcrumb home... view or change your dependents... your va dependents".

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by shiragoodman** [comment]
@joshkimux can you rename the title of this ticket? You can delete all the text (minus the 00.00.1 number) and rename to the issue you found. thanks!

**Comment by joshkimux** [comment]
Done!
